modules/statistics/templates/form_stats_behavioural.tpl

```diff
@@ -97,7 +97,7 @@
             {/foreach}
         </tr>
         <tr>
-            <td class=" pis">Per Instrument Stats</td>
+            <td class=" pis">Per Instrument Data Entry Stats</td>
```

Comment:
```suggestion
            <td class=" pis">Per Instrument Double Data Entry Stats</td>
```
ok, but needs to say *Double* Data Entry stats

Reply:
hang on, is this really the All-Sites link ?  I'll have to go check an instance...

Reply:
@christinerogers it never adds a site id to the query (only the links above in the table do so) but it still appends a project when a project is selected

Reply:
no its not an all sites. it's only the site above in the header